### ISSUE TITLE
how to change the http port from ssh

### ISSUE THREAD

**Issue by joeydream** [ccrisan/motioneyeos#2405]
a Pi zero is installed motioneyeOS and want to connect with other hub.
the http port is changed from 80 to 8081 and it can't be connected by browser amymore

but , it can still can be reached by ssh

can the http port change back to 80 by ssh, nano ? thanks

**Comment by TomasWallflower** [comment]
Yes, Motioeye configration can be changed thru ssh with nano. In file /data/etc/motioneye.conf you need to change row "port". After that you need to reboot target device.
